src/commands/createStaticWebApp/createStaticWebAppFromTemplate.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, ICreateChildImplContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { StaticWebAppTreeItem } from '../../tree/StaticWebAppTreeItem';
+import { SubscriptionTreeItem } from '../../tree/SubscriptionTreeItem';
+import { showSwaCreated } from '../showSwaCreated';
+import { IStaticWebAppWizardContext } from './IStaticWebAppWizardContext';
+
+export async function createStaticWebAppFromTemplate(context: IActionContext & Partial<ICreateChildImplContext> & Partial<IStaticWebAppWizardContext>, node?: SubscriptionTreeItem): Promise<StaticWebAppTreeItem> {
```

Comment:
I think we could refactor this and add it to `createStaticWebApp.ts`.  Kind of similar to how `createStaticWebAppAdvanced`, we could call `createStaticWebApp` and pass in the options to set `context.fromTemplate = true`.

To look cleaner, we could refactor this code:
```
const progressOptions: ProgressOptions = {
        location: ProgressLocation.Notification,
        title: localize('verifyingWorkspace', 'Verifying workspace...')
    };
    await window.withProgress(progressOptions, async () => {
        const folder = await tryGetWorkspaceFolder(context);
        if (folder) {
            await setWorkspaceContexts(context, folder);
        } else {
            await showNoWorkspacePrompt(context);
        }
    });
```
into its own function, and if we see context.fromTemplate === true, we just skip it and go straight down to `node.createChild`.

Another benefit to doing this is we don't have to duplicate the create process at all.  I'm not sure if it was omitted intentionally, but this function is missing the postCreate actions (such as verifying the actions built and opening the node up for the user).  So anytime we wanted to change that behavior, we would have to do it in two places.